code/getkey.h

```diff
@@ -0,0 +1,7 @@
+#ifndef GETKEY_H
+#define GETKEY_H
+
+// Function to get the key pressed.
+char get_key_pressed(void);
+
+#endif
```

Comment:
Have not included get_key() and get_key_window() function into the header file.

try this:-

`#ifndef GETKEY_H
#define GETKEY_H


char get_key_pressed(void);


// Function to get the key pressed system specific.
#if defined(_WIN32) || defined(_WIN64)

    #include<windows.h>
    #include <unistd.h>   //sleep func
    char get_key_window(void);

#else

    #include<ctype.h>
    #include <stdio.h>    //scanf
    char get_key(void);

#endif

#endif`

Reply:
I've read that functions that aren't used outside of the.c file where they're defined shouldn't go in a.h file. That's why i didn't include these functions.

Reply:
Well, for cross platform program its actually a good practice to do it.. To define all the header file and all the functions declarations in the header file.